shell/platform/darwin/ios/framework/Headers/Flutter.h

```diff
@@ -5,6 +5,12 @@
 #ifndef FLUTTER_FLUTTER_H_
 #define FLUTTER_FLUTTER_H_
 
+/**
+ CHANGELOG:
+
+ November 29, 2017: Added a CHANGELOG section.
```

Comment:
is the changelog useful beyond the git history? In my experience, changelogs are always incomplete and rarely that useful...

Reply:
I think this is a better place to put just the breaking changes. The engine Git history if full of messages the user of the public API will never care about.

Reply:
In that case I think we should label this "BREAKING CHANGES", rather than changelog, and we should update the docs for the macros to remind people to add things to this list, and we should update our wiki page to point to this file and say to update it, etc.

Reply:
Done. I am waiting for comments on the doc. But if the suggestions are fine, I will just move it to the wiki.